### PR TITLE
Add a test case for ome/omero-py#418

### PR DESCRIPTION
**Pull request by chris-allan**
# What this PR does

Add a test case for ome/omero-py#418

### CODE REVIEW

File: components/tools/OmeroPy/test/integration/tablestest/test_service.py [hunk 1]
```diff
@@ -714,4 +714,40 @@ def testColumnSpaceNames(self):
         table.delete()
         table.close()
 
+    def tesIncludeRowNumbers(self):
```
Comment (sbesson) [inline components/tools/OmeroPy/test/integration/tablestest/test_service.py]:
```suggestion
    def testIncludeRowNumbers(self):
```
is likely required to be picked up by `pytest`

**Review by sbesson — CHANGES_REQUESTED** [review]
CI execution has been failing overnight due to unrelated infrastructure issues. One syntactic issue to allow the test to be picked up by `pytest`.

Only other wonder if whether we also want to expand the test semantics to ensure that the output

```
table.read(colNumbers, start, stop, context)
table.readCoordinates(rowNumbers, context)
```

also respect the new `omero.tables.include_row_numbers` key/value pair

**Comment by will-moore** [comment]
Code looks good, apart from typo mentioned above.

**Comment by jburel** [comment]
test is green, upstream PR has now been. Requested changes were made to allow the test to be picked up

**Review by sbesson — APPROVED** [review]
All three new integration tests were included in the nightly CI builds, executed successfully and should cover all OMERO.tables API that have support for the new boolean

**Pull request merged**